pkg/servicetest/testdata/services/docker-compose.solr.yaml

```diff
@@ -8,7 +8,9 @@
 #    copy the files from its "solr-conf/6.x" directory into ".ddev/solr/conf"
 #    so that a file exists with the path ".ddev/solr/conf/solrconfig.xml".
 #
-# Note: The configuration files must be present before running `ddev start`.
+# Notes:
+# - The configuration files must be present before running "ddev start".
+# - Changes to the configuration files will not be automatically applied.
```

Comment:
`...even with a ddev rm --data. They must be manually updated inside the docker instance after the initial core creation.`